From 9bcc61944c7cc4a39651bd1569298ef1c1993af9 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 23 May 2023 11:42:38 -0700
Subject: [PATCH] Set chroot group to mockbuild

In order for stateless integration to work properly, we want to specify
an undefined group name ("mockbuild") instead of a defined group name
("mock").

Rebased for mock 2.12 update by Patrick McCarty.
Rebased for mock 4.0 update by William Douglas.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 py/mockbuild/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/py/mockbuild/config.py b/py/mockbuild/config.py
index 88b4737..c7ea1f9 100644
--- a/py/mockbuild/config.py
+++ b/py/mockbuild/config.py
@@ -64,7 +64,7 @@ def setup_default_config_opts():
     config_opts['rpmbuild_timeout'] = 0
     config_opts['chrootuid'] = os.getuid()
     config_opts['chrootgid'] = os.getgid()
-    config_opts['chrootgroup'] = 'mock'
+    config_opts['chrootgroup'] = 'mockbuild'
     config_opts['chrootuser'] = 'mockbuild'
     config_opts['build_log_fmt_name'] = "unadorned"
     config_opts['root_log_fmt_name'] = "detailed"
-- 
2.40.1

